plugins/foreman_openscap/0.5/index.md

```diff
@@ -0,0 +1,220 @@
+  <tr>
+    <td>>= 1.11</td>
+    <td>0.5.0</td>
+    <td>0.5.0</td>
+    <td>0.1.2</td>
+  </tr>
+</table>
+
+# 2. Installation
+
+There are a few components to install:
+
+Foreman OpenSCAP (foreman_openscap), Smart Proxy OpenSCAP (smart_proxy_openscap), foreman_scap_client and puppet-foreman_scap_client.
+
+### 2.1 Installing foreman_openscap
+
+On the Foreman server:
+
+	> yum install ruby193-rubygem-foreman_openscap
+	> service httpd reload
+
+Please refer the Foreman plugin [manual](http://theforeman.org/manuals/latest/index.html#6.1InstallaPlugin) for more information about installing Foreman plugins.
+
+### 2.2 Installing smart_proxy_openscap
+
+On the Smart-Proxy server:
+
+
+    > yum install rubygem-smart_proxy_openscap
+
+Edit ```openscap.yml``` with the appropriate settings
+
+    ---
+    :enabled: true
+    :reportsdir: /path/to/saved/arf_reports/
+    :spooldir: /path/to/arf_reports/which/failed/to/post/to/foreman
+
+
+### 2.3 Installing puppet-foreman_scap_client
+
+    puppet module install isimluk-foreman_scap_client
+
+This puppet module will automatically install foreman_scap_client (if not installed) and will configure the client's ```/etc/foreman_scap_client/config.yaml``` with parameters which are needed for the operation of foreman_scap_client.
+foreman_scap_client to run scans and upload results to the Smart Proxy.
+
+# 4. Usage
+
+This chapter covers features that you can use in terms of Foreman and OpenSCAP
+integration. Everything described below assumes you've sucessfully installed
+foreman_openscap, smart_proxy_openscap and puppet-foreman_scap_client is available on your puppetmaster and Foreman.
+
+*__Please note:__ smart_proxy_openscap is __required__ for the norman operation of foreman_openscap*
+
+You would usually start with uploading SCAP contents, then create policies of those SCAP contents and assign the policy to hosts or hostgroups.
+The puppet module will install ```foreman_scap_client``` and configure it with the needed policy information. The puppet module also adds a cron line,
+which runs the scap client at the schedule select when creating the policy.
+
+## 4.1 Creating SCAP content
+
+foreman_openscap comes with default scap content provided by [scap-security-guide](https://fedorahosted.org/scap-security-guide/).
+
+The default SCAP content is available for RHEL 6, RHEL 7 and Fedora.
+
+Besides the default SCAP content, you can also upload your own SCAP content.
+
+__Access SCAP contents__ - Hosts -> Compliance -> SCAP Contents
+
+__Create SCAP Content__ - You can upload any valid OpenSCAP DataStream file
+
+(After upload, SCAP content is validated at the Smart-Proxy and SCAP profiles are extracted)
+![New SCAP Content]({{page.images}}/create_new_scap_content.png)
+
+
+## 4.2 Creating policy wizard
+
+* Name your policy
+* Choose which SCAP content & SCAP profile to apply
+* Choose schedule when to run this policy
+* Select to which locations / orgaginations this policy belongs to, if enabled
```

Comment:
typo, orgaginations -> organizations

There's a redundant 'to' after select, "Select which locations this policy belongs to"